parse-osmium.cpp

```diff
@@ -21,7 +21,10 @@
 */
 
 #include <boost/format.hpp>
-
+#ifdef _WIN32
+#    define NOMINMAX
+#    include <WinSock2.h>
```

Comment:
Could you explain what error you get here that requires the include? This looks like something that should go into libosmium.